infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/TrustedCAProvisioner.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.provision;
+
+import org.eclipse.che.api.workspace.server.spi.InfrastructureException;
+import org.eclipse.che.workspace.infrastructure.kubernetes.environment.KubernetesEnvironment;
+import org.eclipse.che.workspace.infrastructure.kubernetes.namespace.KubernetesNamespace;
+
```

Comment:
Should it be two separate things? Separate for Che server and separate for the workspaces pods?

Reply:
@skabashnyuk do you have a specific case in mind where it is useful? To me it is a list of trusted CA certificates for Che installation that should be trusted by Che server and components as well as by all workspaces. And this is how it works now for Openshift infrastructure.

Reply:
the use-case is to have all provisioners at the same place. Do you have use case why it should not?